vnext/JSI.Desktop.UnitTests/JSI.Desktop.UnitTests.vcxproj

```diff
@@ -0,0 +1,152 @@
+      <GenerateDebugInformation>DebugFull</GenerateDebugInformation>
+    </Link>
+  </ItemDefinitionGroup>
+  <ItemDefinitionGroup Condition="'$(Configuration)'=='Debug'">
+    <ClCompile>
+      <PreprocessorDefinitions>_DEBUG;DEBUG;%(PreprocessorDefinitions)</PreprocessorDefinitions>
+      <Optimization>Disabled</Optimization>
+      <MinimalRebuild>true</MinimalRebuild>
+      <!-- Use /RTCsu for debug builds. This enables some run-time error
+      checking such as detection of overruns and underruns of local variables
+      and use of uninitialized variables.
+      -->
+      <BasicRuntimeChecks>EnableFastChecks</BasicRuntimeChecks>
+      <RuntimeLibrary>MultiThreadedDebugDLL</RuntimeLibrary>
+    </ClCompile>
+  </ItemDefinitionGroup>
+  <ItemDefinitionGroup Condition="'$(Configuration)'=='Release'">
+    <ClCompile>
+      <PreprocessorDefinitions>NDEBUG;%(PreprocessorDefinitions)</PreprocessorDefinitions>
+      <RuntimeLibrary>MultiThreadedDLL</RuntimeLibrary>
+    </ClCompile>
+    <Link>
+      <OptimizeReferences>true</OptimizeReferences>
+      <EnableCOMDATFolding>true</EnableCOMDATFolding>
+    </Link>
+  </ItemDefinitionGroup>
+  <ItemGroup>
+    <ClInclude Include="..\Desktop.IntegrationTests\TestMessageQueueThread.h" />
+    <ClInclude Include="JsiRuntimeUnitTests.h" />
+  </ItemGroup>
+  <ItemGroup>
+    <ClCompile Include="..\Desktop.IntegrationTests\TestMessageQueueThread.cpp" />
+    <ClCompile Include="ChakraCoreRuntimeUnitTests.cpp" />
+    <ClCompile Include="JsiRuntimeUnitTests.cpp" />
+    <ClCompile Include="RunUnitTests.cpp" />
+  </ItemGroup>
+  <ItemGroup>
+    <None Include="packages.config">
+      <SubType>Designer</SubType>
+    </None>
+  </ItemGroup>
+  <ItemGroup>
+    <ProjectReference Include="..\Folly\Folly.vcxproj">
+      <Project>{A990658C-CE31-4BCC-976F-0FC6B1AF693D}</Project>
+    </ProjectReference>
+    <ProjectReference Include="..\FollyWin32\FollyWin32.vcxproj">
+      <Project>{74085F13-2DDE-45E5-A0CA-927AC9D0B953}</Project>
+    </ProjectReference>
+    <ProjectReference Include="..\ReactCommon\ReactCommon.vcxproj">
+      <Project>{A9D95A91-4DB7-4F72-BEB6-FE8A5C89BFBD}</Project>
+    </ProjectReference>
+    <ProjectReference Include="..\ReactWindowsCore\ReactWindowsCore.vcxproj">
+      <Project>{11C084A3-A57C-4296-A679-CAC17B603144}</Project>
+    </ProjectReference>
+    <ProjectReference Include="..\Desktop\React.Windows.Desktop.vcxproj">
+      <Project>{95048601-C3DC-475F-ADF8-7C0C764C10D5}</Project>
+    </ProjectReference>
+  </ItemGroup>
+  <ItemDefinitionGroup />
+  <Import Project="$(VCTargetsPath)\Microsoft.Cpp.targets" />
+  <ImportGroup Label="ExtensionTargets">
+    <Import Project="$(SolutionDir)packages\ReactWindows.OpenSSL.StdCall.Static.1.0.2-p.2\build\native\ReactWindows.OpenSSL.StdCall.Static.targets" Condition="Exists('$(SolutionDir)packages\ReactWindows.OpenSSL.StdCall.Static.1.0.2-p.2\build\native\ReactWindows.OpenSSL.StdCall.Static.targets')" />
```

Comment:
Does this project use OpenSSL, directly or transitively?

Reply:
This project depends on React.Windows.Desktop.lib, which requires OpenSSL.

---
In reply to: [298775867](https://github.com/microsoft/react-native-windows/pull/2677#discussion_r298775867) [](ancestors = 298775867)